general-concepts/dependencies/text.xml

```diff
@@ -743,8 +743,8 @@ tarball.
 
 <p>
 Packages often have optional dependencies that are needed only when running
-tests. These should be specified in DEPEND behind a USE flag. Often, the
-'test' USE flag is used for this purpose.
+tests. These should be specified in <c>BDEPEND</c> or <c>DEPEND</c> behind a USE flag (see also <uri link="::general-concepts/dependencies/#build-dependencies"/>). Often,
+the <c>test</c> USE flag is used for this purpose.
```

Comment:
Hm, internal cross references are a bit special in the devmanual, as they use the original section title, i.e. instead of `#build-dependencies` it should be `#Build dependencies`. Also avoid lines longer than 80 chars (if at all possible).
```suggestion
tests. These should be specified in <c>BDEPEND</c> or <c>DEPEND</c> behind
a USE flag
(see also <uri link="::general-concepts/dependencies/#Build dependencies"/>).
Often, the <c>test</c> USE flag is used for this purpose.
```